apps/notifications/src/panel/templates/index.jsx

```diff
@@ -375,8 +375,6 @@ class Layout extends Component {
 					 */
 					activateKeyboard();
 					this.props.selectNote( this.state.selectedNote );
-				} else if ( this.props.selectedNoteId ) {
-					this.props.unselectNote();
```

Comment:
This makes it so the left button no longer closes the panel. Was there a conflict we needed to remove this for?

In general, the change should be fine. But note that /read/notifications the panel opens in the opposite direction and there left previously made sense as the closing direction. We will probably want to create a follow up issue to reverse the left/right shortkeys on /read/notifications at some point in the future?